sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -70,43 +73,6 @@ Breaking changes from 1.0.0-preview.5:
 
 ## 1.0.0-preview.4 (2019-10-08)
 
-### Breaking changes from 1.0.0-preview.3:
-
-This release brings our interface a bit closer to what is offered in the
```

Comment:
@praveenkuttappan

Reply:
The added version in line 75 confused our tool. I will update the tooling and manually fix this PR.